src/store/HOCs/withAction.tsx

```diff
@@ -0,0 +1,10 @@
+import { useContext } from "react";
+import { ActionContext } from "../context";
+
+export const withAction = (Component: React.FC) => {
+  const ActionConsumer = (props: unknown) => {
```

Comment:
Tip: Avoid using unknown. Explicitly declare Prop types. Create new types/interfaces under ../types and use those types here